wiki/Gameplay/Sliderbreak/en.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
pretty sure only the first meaning is correct, never heard anyone using the second one

Reply:
was thinking maybe term was used in another mode considering the glossary specifically mentioned osu!standard for the first definition

if not though, i can remove the 2nd definition and rewrite this